src/components/Audiences/index.js

```diff
@@ -21,7 +22,10 @@ const createAudiences = ({ config, logger }) => {
         }
       },
       onResponse(response) {
-        const destinations = response.getPayloadByType("activation:push") || [];
+        const destinations = flatMap(
+          response.getPayloadsByType("activation:push"),
+          payloads => payloads
+        );
```

Comment:
Both of these `flatMap` usages seem like `flat` would be more appropriate, since the mapping is unnecessary. Did you choose not to implement `flat` because you would need to support a depth of greater than `1` to align with the JavaScript spec?

Reply:
Exactly.  I suppose we could create `flatten` or `flat1`, but I thought flatmap would be more understandable and standard.

Reply:
Sounds good. I'm not really opinionated on the matter.